node/cli/src/export_metadata_cmd.rs

```diff
@@ -38,6 +38,8 @@ pub struct ExportMetadataCmd {
 	pub tmp: bool,
 }
 
+// REVIEW: Should this be refactored or panic is okay here?
+#[allow(clippy::unwrap_used)]
```

Comment:
I think panic is OK for in this case; this is only invoked from CLI for development purposes.